app/services/preserved_object_handler.rb

```diff
@@ -118,10 +120,20 @@ def update_version
   private
 
   def moab_validation_errors
+    results = []
     object_dir = "#{storage_location}/#{DruidTools::Druid.new(druid).tree.join('/')}"
     moab = Moab::StorageObject.new(druid, object_dir)
     object_validator = Stanford::StorageObjectValidator.new(moab)
-    object_validator.validation_errors
+    errors = object_validator.validation_errors
+    unless errors.empty?
```

Comment:
this could be `if errors.any?`.   I find unless just slightly harder congnitively.  Okay if you disagree.

Reply:
Ah yeah. I like that. I first did ``` if errors.size > 0 ``` then rubocop told me to change it to ``` unless errors.empty? ``` , this is a better solution ```` if errors.any? ``` !